CHANGELOG.md

```diff
@@ -1,3 +1,10 @@
+# 83.0.1
+
+This patch release resolves some missing content from the theming extension documentation and
```

Comment:
Please add the PR for the fixed docs to this CHANGELOG. Looks like it was missed during generation.

Reply:
The `notes` command doesn't appear to include changes that only affect `docs/` directories.

Reply:
Filed https://github.com/material-components/material-components-ios/issues/7278 to track.